packages/cubejs-schema-compiler/test/unit/fixtures/folders.yml

```diff
@@ -0,0 +1,105 @@
+cubes:
+  - name: orders
+    sql: SELECT * FROM orders
+    joins:
+      - name: users
+        sql: "{CUBE}.order_id = {orders}.id"
+        relationship: many_to_one
+    measures:
+      - name: count
+        sql: id
+        type: count
+    dimensions:
+      - name: id
+        sql: id
+        type: number
+        primary_key: true
+
+      - name: number
+        sql: number
+        type: number
+
+      - name: status
+        sql: status
+        type: string
+
+      - name: city
+        sql: city
+        type: string
+    hierarchies:
+      - name: orders_hierarchy
+        levels:
+          - "{CUBE}.status"
+          - number
+          - users.city
```

Comment:
@vasilev-alex Do we verify that only dimensions can be part of a hierarchy? (And measures/segments can't.)